src/components/PokemonChoice.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './PokemonChoice.css';
+
+
+
+const Modal= (props)=> {
+    return (
+        <div>
+            <div className="modal-wrapper"
+            style={{
+                transform: props.show ? 'translateY(0vh)' : 'translateY(-100vh)',
+                opacity: props.show ? '1' : '0'
+            }}>
```

Comment:
Unless you do javascript animations, toggle `className`s instead of inline styles.